src/utils/getErrorCodeFromResponse.ts

```diff
@@ -0,0 +1,36 @@
+import { DOMParser } from "xmldom";
+
+import { none, Option, some, tryCatch } from "fp-ts/lib/Option";
+
+type SpidError = string;
+
+/**
+ * Extract StatusMessage from SAML response
+ *
+ * ie. for <StatusMessage>ErrorCode nr22</StatusMessage>
+ * returns "22"
+ */
+export default function getErrorCodeFromResponse(
```

Comment:
do not export as default, it makes refactoring code harder (i.e. can't rename the function)